### DISCUSSION TITLE
Possible to jump to line?

### DISCUSSION THREAD

**Discussion by derricck** [zero-plusplus/vscode-operate-from-autohotkey#13] [Q&A]
Hello, is it possible to activate a given open file into the editor and jump to a specific line using some command from the ahk script?

I mean something similar to when you press **CTRL+G** into the vscode and type a line.

Example:
Suppose you have `a.ahk, b.ahk, c.ahk` files, open into vscode, and you currently have focused `c.ahk`, then from your script you call an action that can focus `a.ahk` and jump to the desired line, by focus I mean just set the caret to it.

If it's not possible to 'focus' a specific script only jumping to the line also helps!

**Comment by zero-plusplus** [comment]
It is possible. Specifically, the following API is used for implementation.
* [revealRange](https://code.visualstudio.com/api/references/vscode-api#344)
* [openTextDocument](https://code.visualstudio.com/api/references/vscode-api#3017)

One problem is that this extension does not provide a way to specify command arguments.

In order to implement argument specification, a simple parser would have to be created, which would increase development costs a bit.

I am currently working on other projects and may not be able to implement this right away.

**Reply by derricck** [reply]
I understand, thanks for the answer, if possible let me know if someday you get some time to work on it! :)

**Reply by zero-plusplus** [reply]
OK.
See the following issue for tracking.
* #14
* #15
